public/API/API_GetGame.php

```diff
@@ -22,29 +22,35 @@
  *  string     Released                   release date information for the game
  */
 
+use App\Platform\Models\Game;
+
 $gameID = (int) request()->query('i');
 
-$game = getGameData($gameID);
+$game = Game::with('system')->find($gameID);
+
+if ($game) {
 
-if ($game === null) {
+    $gameData =
+         [
+            'Title' => $game->Title,
+            'GameTitle' => $game->Title,
+            'ConsoleID' => $game->ConsoleID,
+            'ConsoleName' => (string) $game->system->Name,
+            'Console' => (string) $game->system->Name,
+            'ForumTopicID' => $game->ForumTopicID,
+            'Flags' => 0, // Always '0'
+            'GameIcon' => $game->ImageIcon,
+            'ImageIcon' => $game->ImageIcon,
+            'ImageTitle' => $game->ImageTitle,
+            'ImageIngame' => $game->ImageIngame,
+            'ImageBoxArt' => $game->ImageBoxArt,
+            'Publisher' => $game->Publisher,
+            'Developer' => $game->Developer,
+            'Genre' => $game->Genre,
+            'Released' => $game->Released,
+        ];
+
+    return response()->json($gameData);
```

Comment:
local variable can be inlined, and to match indentation:
```suggestion
    return response()->json([
        'Title' => $game->Title,
        'GameTitle' => $game->Title,
        'ConsoleID' => $game->ConsoleID,
        'ConsoleName' => (string) $game->system->Name,
        'Console' => (string) $game->system->Name,
        'ForumTopicID' => $game->ForumTopicID,
        'Flags' => 0, // Always '0'
        'GameIcon' => $game->ImageIcon,
        'ImageIcon' => $game->ImageIcon,
        'ImageTitle' => $game->ImageTitle,
        'ImageIngame' => $game->ImageIngame,
        'ImageBoxArt' => $game->ImageBoxArt,
        'Publisher' => $game->Publisher,
        'Developer' => $game->Developer,
        'Genre' => $game->Genre,
        'Released' => $game->Released,
    ]);
```

Reply:
Alternatively: `$game->only(['Title', 'ConsoleID'])`, then map the few remaining fields, as the response already matches most of those field names.

Reply:
Planned for a long time, it might be worth introducing "Api V1" resource responses: https://laravel.com/docs/10.x/eloquent-resources#resource-responses

Reply:
Using map() adds a few ms to the response time (as I found in the Extended endpoint initially), just `return`ing the data is currenly faster.